cri/metrics/metrics.go

```diff
@@ -13,6 +13,23 @@ func init() {
 	Register()
 }
 
+// Action labels for different pod/image operations
+const (
```

Comment:
Thanks for your contribution. In fact, the pouch's HTTP server also needs  to collect metrics. So could you please to do the samething for api/servers?

Reply:
Done.